Mathlib/AlgebraicGeometry/Morphisms/FiniteType.lean

```diff
@@ -83,16 +83,9 @@ theorem LocallyOfFiniteType.source_openCover_iff {X Y : Scheme.{u}} (f : X ⟶ Y
   locallyOfFiniteType_eq.symm ▸ RingHom.finiteType_is_local.source_openCover_iff f 𝒰
 #align algebraic_geometry.locally_of_finite_type.source_open_cover_iff AlgebraicGeometry.LocallyOfFiniteType.source_openCover_iff
 
-theorem LocallyOfFiniteType.openCover_iff {X Y : Scheme.{u}} (f : X ⟶ Y)
-    (𝒰 : Scheme.OpenCover.{u} Y) :
-    LocallyOfFiniteType f ↔ ∀ i, LocallyOfFiniteType (pullback.snd f (𝒰.map i)) :=
-  locallyOfFiniteType_eq.symm ▸ RingHom.finiteType_is_local.is_local_affineLocally.openCover_iff f 𝒰
-#align algebraic_geometry.locally_of_finite_type.open_cover_iff AlgebraicGeometry.LocallyOfFiniteType.openCover_iff
-
```

Comment:
Can we also get rid of `LocallyOfFiniteType.affine_openCover_iff` and `LocallyOfFiniteType.source_openCover_iff`? Maybe we can introduce some `HasAffineRingProperty` extending `HasAffineProperty` (this can and should of course go in a follow-up PR).

Reply:
Yes that's on my plan but probably next PR.